pkg/crc/api/api_http_test.go

```diff
@@ -126,6 +126,41 @@ func TestHTTPApi(t *testing.T) {
 		},
 		configSetResult,
 	)
+	// Get the cpus config again after setting it to make sure it is set
+	// properly.
+	configGetAfterSetResult, err := client.GetConfig([]string{"cpus"})
+	assert.NoError(t, err)
+	assert.Equal(
+		t,
+		apiClient.GetConfigResult{
+			Success: true,
+			Error:   "",
+			Configs: map[string]interface{}{
+				"cpus": float64(5),
+			},
+		},
+		configGetAfterSetResult,
+	)
+
+	configUnsetResult, err := client.UnsetConfig([]string{"cpus"})
```

Comment:
Can there also be a test for getting multiple config values?

Reply:
@cfergeau we can add more test.